tslint.prod.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": ["./tslint.json"],
+  "defaultSeverity": "error",
+  "rules": {
+    "no-debugger": true,
+    /**
+    * There is a strange issue when using no-unused-variable as part of the react build, so we are using it as separate
+    * tslint call, so please don't put that in tslint.json unless below issue gets fixed.
+    * https://github.com/Realytics/fork-ts-checker-webpack-plugin/issues/74
+    */
```

Comment:
Does this work?
It's red in GitHub... And I recall that these comments are invalid in JSON.

Reply:
I have not seen comments in JSON (which is a huge downfall).
Did something change to allow comments?

Reply:
There are no complaints in processing the JSON comments so the processor that webpack uses is cool with it.

Reply:
That's right, comments are not really supported by JSON, but preprocesors take care of stripping them before actually using, which is win :)